File: Organizer.Model/Entities/Movement.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Organizer.Model
{
    [Table("Movement")]
    public class Movement
    {
        [Key]
        [Column("MovementId")]
        public long Id { get; set; }

        public MovementType Type { get; set; }

        public MovementOrigin Origin { get; set; }

        public long? OriginAccountId { get; set; }

        public Account OriginAccount { get; set; }

        public string Description { get; set; }

        public long CurrencyId { get; set; }

        public Currency Currency { get; set; }

        public decimal Amount { get; set; }

        public DateTime Date { get; set; }

    }
}

File: Organizer.Loader/src/entities/movement.ts
import { MovementType } from "entities/movementType";
import { Currency } from "entities/currency";
import { Account } from "entities/account";
import { MovementOrigin } from "entities/movementOrigin";

export class Movement {
    public Id: number;
    public Type: MovementType;
    public Origin: MovementOrigin;
    public OriginAccountId: number;
    public Description: string;
    public CurrencyId: number;
    public Amount: number;
    public Date: Date;
}